### ISSUE TITLE
Equalise function evaluation cost on Mantid data

### ISSUE THREAD

**Issue by tyronerees** [fitbenchmarking/fitbenchmarking#275]
**Description of the error**
A function evaluation of native Mantid data is currently much faster when fitting using Mantid than any other solver.  

**Describe the expected result**
One function evaluation should take the same time regardless of the minimizer used.

**Describe the actual result**
As outlined in https://github.com/fitbenchmarking/fitbenchmarking/pull/263#issuecomment-542158032:
> Regarding the timings -- after a lot of digging, it seems that the way Mantid calls Mantid models in Master was orders of magnitude faster than when Mantid models are called by the other minimizers -- timing a call to `eval_f` shows that takes ~0.05s, whereas previously Mantid took only
> ~0.03s to return using a method that took at least 500 function evaluations.

**Suggested fix**
Amend function evaluations to remove overhead in third party solvers due to, e.g., using  numpy arrays instead of mantid workspaces.
Or, possibly, make Mantid use the same process so that a function evaluation costs the same (if slower)

**Comment by tyronerees** [comment]
@AnthonyLim23 -- do you think this can be closed now?

**Comment by AnthonyLim23** [comment]
yes and no. I have reduced the bias, but it still exists (see the slides on isolate_bias). We have quantified it now too, so that's something.
